.github/workflows/sycl-rel-nightly.yml

```diff
@@ -12,8 +12,8 @@ jobs:
     with:
       build_cache_root: "/__w/"
       build_artifact_suffix: default
-      build_configure_extra_args: '--disable-jit --no-assertions --add_security_flags=sanitize --hip --cuda'
-      build_image: ghcr.io/intel/llvm/ubuntu2204_build:latest-133fee559371ce0e6ff867e378c21cde2bdf6c90
+      build_configure_extra_args: '-DCMAKE_INSTALL_LIBDIR=lib --disable-jit --no-assertions --add_security_flags=sanitize --hip --cuda'
+      build_image: ghcr.io/kornevnikita/llvm:latest
```

Comment:
imo we cant/shouldn't use images from forks. can you add this image to this repo?

Reply:
Uploaded to intel/llvm. Test is here https://github.com/intel/llvm/actions/runs/16032068467. I believe the result is fine, just windows runner & cuda are making their business.